### PR TITLE
NISRA Covid changes December 2020

### PR DESCRIPTION
**Pull request by JamesGardiner**
### What is the context of this PR?

Update NISRA HH/HI schemas with recent Covid changes

### How to review

Check new content is correct as specified at https://docs.google.com/spreadsheets/d/1NHfR91ZyfoJiafGrX6ZzySSifZLeA4gi8Uxp1NZsTnA/edit#gid=2097876145 (tab should be `NISRA Changes Dec20`

### Checklist

- [x] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra_changes_dec_2020/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra_changes_dec_2020/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

File: source/jsonnet/northern-ireland/individual/blocks/employment/employment_status_last_seven_days.jsonnet [hunk 1]
```diff
@@ -63,17 +64,20 @@ local proxyTitle = {
   ],
 };
 
+local nonProxyDescription = 'If you have a job but have been off work on <em>furlough</em>, select “Temporarily away from work ill, on holiday or temporarily laid off”';
```
Comment (berroar) [inline source/jsonnet/northern-ireland/individual/blocks/employment/employment_status_last_seven_days.jsonnet:67]: We should follow the style guide and use a function for these descriptions.

Comment (JamesGardiner) [reply source/jsonnet/northern-ireland/individual/blocks/employment/employment_status_last_seven_days.jsonnet]: Done in #d1a750d

File: source/jsonnet/northern-ireland/individual/blocks/employment/employment_status_last_seven_days.jsonnet [hunk 2]
```diff
@@ -55,25 +56,31 @@ local question(title) = {
...
   page_title: 'Employment status in the last seven days',
   question_variants: [
     {
-      question: question(nonProxyTitle),
+      question: question(questionTitle(false), questionDescription(false)),
       when: [rules.isNotProxy],
     },
     {
-      question: question(proxyTitle),
+      question: question(questionTitle(true), questionDescription(true)),
```
Comment (ajmaddaford) [inline source/jsonnet/northern-ireland/individual/blocks/employment/employment_status_last_seven_days.jsonnet:83]: An `isProxy` variable should be passed to `question`, and the `questionTitle` and `questionDescription` methods called from there. See https://github.com/ONSdigital/eq-questionnaire-schemas/blob/master/source/jsonnet/england-wales/individual/blocks/employment/hours_worked.jsonnet for an example.

**Review by ajmaddaford — APPROVED** [review]

**Review by petechd — COMMENTED** [review]
I think we only use **em** tags in titles, are we not supposed to use **strong** in descriptions?

**Comment by JamesGardiner** [comment]
> I think we only use **em** tags in titles, are we not supposed to use **strong** in descriptions?

I feel like I've had this conversation before Pete... I can see instances of both throughout the code. @ajmaddaford do you know the correct answer to this one?

**Comment by JamesGardiner** [comment]
@petechd as discussed offline, there is a deprioritised card in tech backlog that covers a review of strong/em tags in schemas. Until then, using either is fine 👍🏻

**Comment by petechd** [comment]
> @petechd as discussed offline, there is a deprioritised card in tech backlog that covers a review of strong/em tags in schemas. Until then, using either is fine 👍🏻

Ok, otherwise this PR looks good!

**Review by petechd — APPROVED** [review]

**Pull request merged**